pairAndIntroduce.ts

```diff
@@ -9,5 +24,38 @@ const app = new App({
   appToken: process.env.SLACK_APP_TOKEN // add this
 });
 
-const pairs = generatePairs(app);
+export const startConversations = async (app: App) => {
```

Comment:
i wonder if this method should just be moved into it's own file in the `utils` folder and just call the method here instead